catavatar/catavatar.php

```diff
@@ -142,15 +152,15 @@ function catavatar_addon_settings_post(&$a, &$s) {
  * @param $a array
  * @param &$b array
  */
-function catavatar_lookup($a, &$b) {
-	$user = dba::selectFirst('user', ['uid'],['email'=>$b['email']]);
-	
-	$url = $a->get_baseurl().'/catavatar/'.$user['uid'];
+function catavatar_lookup($a, &$b)
```

Comment:
Obsolete: Please use the following argument list instead: `(\Friendica\App $a, &$b)`